cli/src/cli.rs

```diff
@@ -93,6 +137,18 @@ pub enum Commands {
         command: AuthCommands,
     },
 
+    /// Get detailed information about a resource
+    Get {
+        #[clap(subcommand)]
+        resource: GetResourceType,
+    },
```

Comment:
We should decide if we want `sm get resource` or `sm resource get`. I did the latter one and have implemented it for releases and devices, but I think either is fine, just need to decide.
At least I don't think we should have both `sm devices ls` and `sm get devices`